dali/pipeline/operator/builtin/external_source.h

```diff
@@ -197,39 +214,38 @@ class ExternalSource : public Operator<Backend>, virtual public BatchSizeProvide
   /**
    * @brief Sets the data that should be passed out of the op on the next iteration.
    */
-  template<typename SrcBackend>
+  template <typename SrcBackend>
   inline void SetDataSource(const TensorList<SrcBackend> &tl, cudaStream_t stream = 0,
-                            bool sync = false, bool use_copy_kernel = false) {
+                            SetDataConfig set_data_config = {}) {
```

Comment:
Naming this var `set_data_config` kind of suggests, that it's a function - we sometimes use snake_case for functions and the `set` prefix really indicates a function. Maybe renaming it, as I suggested above?

Reply:
We also use *only* snake case for variable names, especially function parameters.